pkg/cmd/secret/list/list_test.go

```diff
@@ -194,9 +197,9 @@ func Test_listRun(t *testing.T) {
 				UserSecrets: true,
 			},
 			wantOut: []string{
-				"SECRET_ONE\t1988-10-11\t",
-				"SECRET_TWO\t2020-12-04\t",
-				"SECRET_THREE\t1975-11-30\t",
+				"SECRET_ONE\t1988-10-11\tSELECTED",
+				"SECRET_TWO\t2020-12-04\tSELECTED",
+				"SECRET_THREE\t1975-11-30\tSELECTED",
```

Comment:
how come this expected output changed?

Reply:
Previously the output lines assertion was matching on partial strings. I confirmed that the `SELECTED` column was there before my changes, the `test.ExpectLines` assertion used regex and the first part of the line matched the expected output so it passed.

Reply:
I need guide line please because am very new to this and I need support to make issues I don't understand.